lib/ansible/plugins/inventory/kubevirt.py

```diff
@@ -0,0 +1,178 @@
+# Copyright (c) 2018 Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+
+__metaclass__ = type
+
+DOCUMENTATION = '''
+    name: kubevirt
+    plugin_type: inventory
+    author:
+      - KubeVirt Team (@kubevirt)
+
+    short_description: KubeVirt inventory source
+
+    description:
+      - Fetch VirtualMachines and VirtualMachinesInstances for one or more clusters
+      - Groups by cluster name, namespace, namespace_vms, namespace_vmis  and labels
+      - Uses kubevirt.(yml|yaml) YAML configuration file to set parameter values.
+
+    options:
+      plugin:
+        description: token that ensures this is a source file for the 'kubevirt' plugin.
+        required: True
+        choices: ['kubevirt']
+      connections:
```

Comment:
I see that there is one more param in lookup params [here](https://docs.ansible.com/ansible/2.5/plugins/lookup/k8s.html) which is missing from the list. It is `kind` parameter. Should we add it here?

Reply:
That's lookup plugin not inventory plugin. It's not relevant here.